client/src/helpers/APICalls/request.ts

```diff
@@ -0,0 +1,55 @@
+import { RequestApiData, RequestStatusApiData } from '../../interface/Requests';
+import { FetchOptions } from '../../interface/FetchOptions';
+
+export const createRequest = async (sitterId: any, startDate: number, endDate: number): Promise<RequestApiData> => {
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({ sitterId, startDate, endDate }),
+    credentials: 'include',
+  };
+
+  try {
+    const fetchData = await fetch(`/request/create`, fetchOptions);
+    const createResponse = await fetchData.json();
+    return createResponse;
+  } catch (e) {
+    return { error: { message: 'Unable to connect to server. Please try again' } };
```

Comment:
I think it would be nice to return the actual error message as opposed to generic text